tests/browser/nav-cookie.browser.js

```diff
@@ -5,9 +5,9 @@
 
 import test from '../../tools/jil/browser-test'
 import { setup } from './utils/setup'
-import { setConfiguration } from '../../packages/browser-agent-core/common/config/config'
-import * as navCookie from '../../packages/browser-agent-core/common/cookie/nav-cookie'
-import * as startTime from '../../packages/browser-agent-core/common/timing/start-time'
+import { setConfiguration } from '../../dist/packages/browser-agent-core/src/common/config/config'
```

Comment:
is there a preference to use the core files from dist over importing from their source path?

Reply:
For right now I think it best for imports inside the tests directory to hit the dist files. When we start converting these files to TS, im not sure browserify is going to like it.